lib/msal-browser/src/config/Configuration.ts

```diff
@@ -78,6 +79,7 @@ export type BrowserSystemOptions = SystemOptions & {
     redirectNavigationTimeout?: number;
     asyncPopups?: boolean;
     allowRedirectInIframe?: boolean;
+    refreshTokenBinding?: boolean;
```

Comment:
Given this PR doesn't ship the entire bound RT feature (ie enabling this flag wont result in a bound RT), I think it would be good if this flag was not exposed here and instead exposed in the final PR.

Reply:
I would propose creating a new temp/staging branch off of dev, and retarget your PRs to that branch, and then when the feature is ready e2e, open a new PR from that branch to dev (which should just require simple sign off instead of a full review).

Reply:
Added an intermediary branch called refresh-token-binding to merge into